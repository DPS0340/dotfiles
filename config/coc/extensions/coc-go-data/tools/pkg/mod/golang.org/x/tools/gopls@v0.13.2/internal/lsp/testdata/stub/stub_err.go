package stub

func main() {
	var br error = &customErr{} //@suggestedfix("&", "quickfix", "")
}

type customErr struct{}
