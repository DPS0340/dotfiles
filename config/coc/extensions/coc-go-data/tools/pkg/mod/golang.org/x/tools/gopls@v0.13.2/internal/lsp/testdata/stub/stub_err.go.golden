-- suggestedfix_stub_err_4_17 --
package stub

func main() {
	var br error = &customErr{} //@suggestedfix("&", "quickfix", "")
}

type customErr struct{}

// Error implements error.
func (*customErr) Error() string {
	panic("unimplemented")
}

